tests/flow.rs

```diff
@@ -613,22 +614,136 @@ fn test_blind_sign_request() -> CredxResult<()> {
         &schema_claims,
     )?;
 
-    let (issuer_public, mut issuer) = Issuer::<BbsScheme>::new(&cred_schema);
+    let (issuer_public_1, mut issuer_1) = Issuer::<BbsScheme>::new(&cred_schema);
+    let (issuer_public_2, mut issuer_2) = Issuer::<BbsScheme>::new(&cred_schema);
+
+    let blind_claims_1 = btreemap! { "link_secret".to_string() => ScalarClaim::from(Scalar::random(rand_core::OsRng)).into() };
+
+    let mut nonce = [0u8; 16];
+    rand::thread_rng().fill_bytes(&mut nonce);
 
-    let blind_claims = btreemap! { "link_secret".to_string() => ScalarClaim::from(Scalar::random(rand_core::OsRng)).into() };
-    let (request, blinder) = BlindCredentialRequest::new(&issuer_public, &blind_claims)?;
+    // equal link_secret equality claims
+
+    let res = check_link_secret_equality(
+        CRED_ID,
```

Comment:
Normally you don't use the same CRED_ID from two different issuers. Each issuer would use their own CRED_ID.

Reply:
I updated the code to use different ids for each credential.